src/helpers/momentHelper.js

```diff
@@ -16,6 +16,14 @@ export const momentFormat = (
   dateTimeFormat = 'YYYY-MM-DD HH:mm:ss Z'
 ) => moment(dateTime, dateTimeFormat).format(returnFormat);
 
+/**
+ * @param {string} date Date as utc string
+ * @param {string} [returnFormat] format to return in
+ * @returns formated date to given format
+ */
+export const momentFormatUtcToLocal = (date, returnFormat = 'DD.MM.YYYY') =>
```

Comment:
`date`? isn't it `dateTime` as well as for the existing `momentFormat` helper. espacially if the format is fix to `'YYYY-MM-DD HH:mm:ss Z'`